app/Providers/AppServiceProvider.php

```diff
@@ -21,6 +22,29 @@ public function register(): void
      */
     public function boot(): void
     {
-        //
+        UploadedFile::macro('makeFromUrl', function (string $url): ?UploadedFile {
+            $tempFile = tempnam(sys_get_temp_dir(), 'pest');
```

Comment:
Why use `pest` here?

Reply:
thanks for reminding me I forgot to change...

I named it while I was building this macro for testing and forgot to change it 🫡